apps/guide/src/content/03-popular-topics/01-frequently-asked-questions.mdx

```diff
@@ -0,0 +1,498 @@
+
+<CH.Code>
+
+```js
+client.user.setUsername('username');
+```
+
+</CH.Code>
+
+### How do I set my bot's avatar?
+
+<CH.Code>
+
+```js
+client.user.setAvatar('URL or path');
+```
+
+</CH.Code>
+
+### How do I set my playing status?
+
+<CH.Code>
+
+```js
+client.user.setActivity('activity');
+```
+
+</CH.Code>
+
+### How do I set my status to "Watching/Listening to/Competing in ..."?
+
+<CH.Code>
+
+```js
+import { ActivityType } from 'discord.js';
+
+client.user.setActivity('activity', { type: ActivityType.Watching });
+client.user.setActivity('activity', { type: ActivityType.Listening });
+client.user.setActivity('activity', { type: ActivityType.Competing });
+```
+
+</CH.Code>
+
+<Alert title="Tip" type="info">
+	If you would like to set your activity upon startup, you can use the _`ClientOptions`_ object to set the appropriate
+	_`Presence`_ data.
+</Alert>
+
+### How do I make my bot display online/idle/dnd/invisible?
+
+<CH.Code>
+
+```js
+client.user.setStatus('online');
+client.user.setStatus('idle');
+client.user.setStatus('dnd');
+client.user.setStatus('invisible');
+```
+
+</CH.Code>
+
+### How do I set both status and activity in one go?
+
+<CH.Code>
+
+```js
+client.user.setPresence({ activities: [{ name: 'activity' }], status: 'idle' });
+```
+
+</CH.Code>
+
+## Miscellaneous
+
+### How do I send a message to a specific channel?
+
+<CH.Code>
+
+```js
+const channel = client.channels.cache.get('id');
+channel.send('content');
+```
+
+</CH.Code>
+
+### How do I create a post in a forum channel?
+
+<Alert title="Tip" type="info">
+	Currently, the only way to get tag ids is programmatically through{' '}
+	<DocsLink type="class" parent="ForumChannel" symbol="availableTags" />.
+</Alert>
+
+<CH.Code>
+
+```js
+const channel = client.channels.cache.get('id');
+
+channel.threads.create({
+	name: 'Post name',
+	message: { content: 'Message content' },
+	appliedTags: ['tagId', 'anotherTagId'],
+});
+```
+
+</CH.Code>
+
+### How do I DM a specific user?
+
+<CH.Code>
+
+```js
+client.users.send('id', 'content');
+```
+
+</CH.Code>
+
+<Alert title="Tip" type="info">
+	If you want to send a direct message to the user who sent the interaction, you can use _`interaction.user.send()`_.
+</Alert>
+
+### How do I mention a specific user in a message?
+
+<CH.Code>
+
+```js
+const user = interaction.options.getUser('target');
+await interaction.reply(`Hi, ${user}.`);
+await interaction.followUp(`Hi, <@${user.id}>.`);
+```
+
+</CH.Code>
+
+<Alert title="Tip" type="info">
+	Mentions in embeds may resolve correctly in embed titles, descriptions and field values but will never notify the
+	user. Other areas do not support mentions at all.
+</Alert>
+
+### How do I control which users and/or roles are mentioned in a message?
+
+Controlling which mentions will send a ping is done via the _`allowedMentions`_ option, which replaces _`disableMentions`_.
+
+This can be set as a default in _`ClientOptions`_, and controlled per-message sent by your bot.
```

Comment:
Could we DocsLink this?

Reply:
Done with 14f37e9f2c061f4a0547c02f917e99a7716c31d0!